_data/investing.yml

```diff
@@ -33,6 +33,12 @@ websites:
       exceptions:
           text: "2FA only available on investment accounts. Not available for 401K plans."
 
+    - name: HealthEquity
+      url: https://www.healthequity.com/
```

Comment:
Yeah http://ir.healthequity.com/ works but not www.healthequity.com. Odd, U.S. IP and all.